src/shared/components/ShareWithColleagues/ShareWithColleagues.types.ts

```diff
@@ -4,6 +4,12 @@ export enum ContributorInfoRights {
 	OWNER = 'OWNER',
 }
 
+export enum ContributorInfoRightsInDutch {
+	CONTRIBUTOR = 'Bijdrager',
+	VIEWER = 'Bekijker',
+	OWNER = 'Eigenaar',
+}
```

Comment:
All labels should be translatable
Always wrap translations in a function, so the app has a chance to fetch the translations from the server, before the tText functions are executed.
```suggestion
export function GetContributorRightLabels() {
	return {
		CONTRIBUTOR = tText('Bijdrager'),
		VIEWER = tText('Bekijker'),
		OWNER = tText('Eigenaar'),
	}
}
```